jinahub/indexers/indexer/LMDBIndexer/__init__.py

```diff
@@ -67,10 +67,11 @@ def update(self, docs: DocumentArray, parameters: Dict, **kwargs):
         :param docs: the documents to update
         :param parameters: parameters to the request
         """
-        trav_path = parameters.get('traversal_path', self.default_traversal_path)
+        trav_paths = parameters.get('traversal_paths', self.default_traversal_paths)
         with self.env.begin(write=True) as t:
```

Comment:
Can we get a better variable name for the `t`? Thanks :)

Reply:
done